flink-runtime/src/main/java/org/apache/flink/runtime/dispatcher/runner/DispatcherRunner.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.dispatcher.runner;
+
+import org.apache.flink.runtime.dispatcher.Dispatcher;
+import org.apache.flink.util.AutoCloseableAsync;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * The {@link DispatcherRunner} encapsulates how a {@link Dispatcher} is being executed.
+ */
+public interface DispatcherRunner extends AutoCloseableAsync {
+
+	/**
+	 * Get the currently running {@link Dispatcher}.
+	 *
+	 * @return the currently running dispatcher or null if none is running
+	 */
+	Dispatcher getDispatcher();
```

Comment:
`@Nullable` is missing

Reply:
I forgot to update the JavaDoc. The return value should always be non-null. Will do it instead of `@Nullable`.

Reply:
ok